.travis.yml

```diff
@@ -1,6 +1,8 @@
 language: scala
 scala:
   - 2.11.7
+jdk:
+  - oraclejdk8
```

Comment:
I was following some instructions on Travis' blog about getting Java 8 code to compile.